src/Microsoft.DotNet.ProjectJsonMigration/Rules/MigrateBuildOptionsRule.cs

```diff
@@ -355,5 +355,15 @@ private IncludeContext GetCopyToOutputIncludeContext(CommonCompilerOptions compi
                     null,
                     ProjectFilesCollection.DefaultPublishExcludePatterns);
         }
+
+        private string FormatLanguageVersion(string langVersion)
+        {
+            if (langVersion.StartsWith("csharp"))
```

Comment:
What about case sensitivity? If `CSHARP5` is in the project.json, does `dotnet build` still do the right thing?